todo_app/app.py

```diff
@@ -17,18 +19,21 @@ def index():
     def create_new_task():
         task_name = request.form.get("name")
         task_description = request.form.get("description")
+        logInfo(f'Adding new task to DB with name: {task_name}, description: {task_description}')

```

Comment:
Worth considering whether this is likely to catch you out with personal data at some point - users are bound to use people's names in their tasks, and very plausibly things like:
* Update my bank details with <list of accounts>
* Rotate all my passwords from `chelsea123`
* Send bob@email.com a message to say hi

So this might be something where we want to avoid logging anything too user-facing here